packages/analytics/analytics-chart/src/components/AnalyticsChart.vue

```diff
@@ -364,10 +377,6 @@ const hasValidChartData = computed(() => {
   return props.chartData && props.chartData.meta && props.chartData.data.length
 })
 
-const showChartHeader = computed(() => {
```

Comment:
If a chart doesn't have a title and doesn't have valid data, does this behave the same way as before?  (It seems like we're showing a header in that case where before we didn't.)

Reply:
Yes, because even in those cases, we may still want to show the kebab menu that may still contain valid actions a user can take, despite the lack of valid chart data. 

For example, a chart in a dashboard may not have data for whatever reason. A user may still want to "go to explore" or "edit" the tile in order to investigate why the chart has no data, and make changes. So I think the old way has to be changed.

Reply:
I think that makes sense; my main concern is that this is changing outside of the feature flag.  I took a look at the sandbox, and I think the change is basically invisible, so I won't worry about it.